e2e_test/test_smoketest.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eo pipefail
+
+source shared.sh
+prepare_node
+
+(cd debug-fee-currency && forge build .)
+export COMPILED_TEST_CONTRACT=../debug-fee-currency/out/DebugFeeCurrency.sol/DebugFeeCurrency.json
+cd js-tests && ./node_modules/mocha/bin/mocha.js test_viem_smoketest.mjs --exit
```

Comment:
We could add a default timeout of 10 seconds here (`--timeout 10000`) if we don't prefer the explicitness of having it on every test.

Reply:
Nice, thanks!